src/content/en/2020/javascript.md

```diff
@@ -272,9 +301,9 @@ Minification is a great way to help reduce file size, but compression is even mo
   sql_file="compression_method.sql"
 ) }}
 
-85% of all JavaScript requests have some level of network compression applied. Gzip makes up the majority of that, with 65% of scripts having Gzip compression applied compared to 20% for Brotli. While the percentage of Brotli (which is more effective than Gzip) is low compared to its browser support, it's trending in the right direction, increasing by 5 percentage points in the last year.
+85% of all JavaScript requests have some level of network compression applied. Gzip makes up the majority of that, with 65% of scripts having gzip compression applied compared to 20% for Brotli (br). While the percentage of Brotli (which is more effective than gzip) is low compared to its browser support, it's trending in the right direction, increasing by 5 percentage points in the last year.
```

Comment:
Hmm, I thought that spelling "Gzip" with a capital letter was correct. Because during the Compression chapter edits this spelling was used (see https://github.com/HTTPArchive/almanac.httparchive.org/pull/1688/commits/3b475a13c9fab10679a6b11158e6b8dfe8c9711c). Just want to achieve consistency between chapters.

Reply:
Good catch. I did a search and found most usage to be lowercase. We should be consistent between chapters but now I'm not sure which is more correct.

Reply:
GNU says Gzip: https://www.gnu.org/software/gzip/

I say Gzip for format and `gzip` for command and file eztension.